src/Core/Checkout/Customer/SalesChannel/AccountService.php

```diff
@@ -90,7 +96,7 @@ public function login(string $email, SalesChannelContext $context, bool $include
      * @throws BadCredentialsException
      * @throws CustomerNotFoundByIdException
      */
-    public function loginById(string $id, SalesChannelContext $context): string
+    public function loginById(string $id, SalesChannelContext $context, ?string $userId = null): string
```

Comment:
Modifying the signature of a public method is not allowed.  I would hide this maybe with an extension inside $context

Reply:
Removed the extra parameter inhttps://github.com/shopware/shopware/pull/3727/commits/65ddefacfe73f7c590d4acfed239fc8279cc16c4